schema/rdls_schema.json

```
@@ -1484,7 +1484,7 @@
         "coordinates": {
           "title": "Coordinates",
           "type": "array",
-          "description": "One or more GeoJSON positions according to the GeoJSON geometry type defined in `.type`.",
+          "description": "One or more GeoJSON positions according to the GeoJSON geometry type defined in `.type`. Values must be given in pairs using the World Geodetic System 1984 (WGS 84) [WGS84] datum, with longitude and latitude units of decimal degrees. E.g. (51.751944,-1.257778) for a 'point` `.type`. Simplified geometries are recommended.",
```

Align description with GeoJSON specification. Remove example because separators differ in JSON and spreadsheet format. Remove simplified geometry recommendation because the concern raised in https://github.com/GFDRR/rdl-standard/issues/197#issuecomment-1687898659 is better addressed by adding the following sentence to the description of `Location.geometry`:

>  This field describes the extent of the geographical area. To avoid creating very large metadata, geometries should have fewer than 50 vertices.

```suggestion
          "description": "One or more GeoJSON positions according to the GeoJSON geometry type defined in `.type`. Each position is an array of numbers containing exactly two numbers with longitude (easting) first and latitude (northing) second. Coordinates must be specified using the World Geodetic System 1984 (WGS 84) datum, with longitude and latitude units of decimal degrees. This is equivalent to the coordinate reference system identified by the Open Geospatial Consortium URN urn:ogc:def:crs:OGC::CRS84.",
```

---

`Location.geometry` still needs to be updated.